src/planner/operator/ddl/logical_alter.cpp

```diff
@@ -0,0 +1,45 @@
+#include "planner/operator/ddl/logical_alter.h"
+
+namespace kuzu {
+namespace planner {
+
+std::string LogicalAlterPrintInfo::toString() const {
```

Comment:
User LCOV_EXCL_START & LCOV_EXCL_STOP to escape code coverage checking from l6-45. U might need to apply this to all toString functions.